### PR TITLE
added google-client, code to module and js file

### PR DESCRIPTION
**Pull request by priya-sudarshanam**
@Richard - 
did the following tasks:
a)added google_client folder to the repository
b)added code to the .module and .js.tpl.php files
Can you pls check them?

thanks

### CODE REVIEW

**Comment by Ferdi** [comment]
@priya-sudarshanam thanks. you should not commit the .svn directory

File: ossap_stats/google-api-php-client/.svn/entries [hunk 1]
```diff
@@ -0,0 +1 @@
+12
```
Comment (rbran100) [inline ossap_stats/google-api-php-client/.svn/entries]: No need to checkin the .svn files, you can download a clean version using "svn export", removing the .svn directories will also work

**Comment by rbran100** [comment]
@priya-sudarshanam @wuqiank do you mind separating the external library and the module code into two separate PR's

You can remove os_board from the make file here: https://github.com/openscholar/openscholar/blob/SCHOLAR-3.x/openscholar/os-ops.make#L14
as well?

File: ossap_stats/google-api-php-client/.svn/entries (hunk 1 above)
Comment (priya-sudarshanam) [inline ossap_stats/google-api-php-client/.svn/entries]: Hi Richard, 
  Just removed the .svn directories. 

Thanks,
Priya
From: Richard Brandon [notifications@github.com]Sent: Wednesday, April 30, 2014 2:13 PMTo: openscholar/ossapCc: Sudarshanam, PriyashreeSubject: Re: [ossap] added google-client, code to module and js file (#22)

In ossap_stats/google-api-php-client/.svn/entries:

> @@ -0,0 +1 @@
> +12

No need to checkin the .svn files, you can download a clean version using "svn export", removing the .svn directories will also work
—
Reply to this email directly or 
view it on GitHub.

**Comment by priya-sudarshanam** [comment]
Hi Ferdi,
  Just removed it. Thanks for letting me know.

Thanks,
Priya
From: Ferdinand Alimadhi [notifications@github.com]Sent: Tuesday, April 29, 2014 4:42 PMTo: openscholar/ossapCc: Sudarshanam, PriyashreeSubject: Re: [ossap] added google-client, code to module and js file (#22)

@priya-sudarshanam thanks. you should not commit the .svn directory
—
Reply to this email directly or 
view it on GitHub.

**Comment by priya-sudarshanam** [comment]
@rbran100 
Only code

File: ossap_stats/ossap-stats-embed-js.tpl.php [hunk 2]
```diff
@@ -63,7 +75,7 @@
     <?php $value = ($key == 'filesize_bytes') ? format_size($value) : $value ?>
     <?php $value = ($key != "os_version" && is_numeric($value)) ? number_format($value) : $value ?>
     <?php $value = ($key == 'activity-messages') ? $value : "'{$value}'" ?>
-    elements[i].innerHTML = <?php echo $value; ?>;
+	elements[i].innerHTML = <?php echo $value; ?>;
```
Comment (rbran100) [inline ossap_stats/ossap-stats-embed-js.tpl.php]: Can you replace this tab with spaces.

File: ossap_stats/ossap_stats.module [hunk 3]
```diff
@@ -31,10 +31,139 @@ function ossap_stats_menu() {
...
+}
+
+/*
+* include all the required files via module_load_include
+* 
+*/
+
+module_load_include('php', 'ossap_stats', 'google-api-php-client/src/Google_Client');
+module_load_include('php', 'ossap_stats', 'google-api-php-client/src/contrib/Google_AnalyticsService');
+module_load_include('inc', 'ossap_stats', 'ossap_stats_credentials');
```
Comment (rbran100) [inline ossap_stats/ossap_stats.module]: Can you move this to either a separate function like before or inline into the function where it is used.  This should not be included all of the time but only when needed

File: ossap_stats/ossap_stats.module [hunk 4]
```diff
@@ -46,6 +175,9 @@ function ossap_stats_theme() {
         'os_version' => NULL,
         'messages' => NULL,
         'getsatisfaction' => NULL,
+		'os_board_visited_pages' => NULL,
+		'os_board_viewed_pages' => NULL
```
Comment (rbran100) [inline ossap_stats/ossap_stats.module]: tabs

**Comment by priya-sudarshanam** [comment]
@rbran100 
a)made the module_load_include as a separate function
b)cleared the tabs in js.tpl and .module files

**Comment by priya-sudarshanam** [comment]
@rbran100 
Function name was misspelt, so fixed it and creating a new PR

**Comment by rbran100** [comment]
Thanks Pria

File: ossap_stats/ossap_stats.module [hunk 5]
```diff
@@ -31,10 +31,144 @@ function ossap_stats_menu() {
...
+
+  return system_settings_form($form);
+
+}
+
+/*
+* include all the required files via module_load_include
+* 
+*/
+function load_includes(){
```
Comment (rbran100) [inline ossap_stats/ossap_stats.module]: Can you namespace this function with os_board please.

File: ossap_stats/ossap_stats_credentials.inc [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+		return $results;	
+}
+
+/* 
+* return_pages
+* no parameters 
+* taps into google analytics and retrieves the results for 
+* pages that are currently active
+*/
+function return_pages(){
```
Comment (rbran100) [inline ossap_stats/ossap_stats_credentials.inc]: Can you namespace this function as well it is too vague.

**Comment by priya-sudarshanam** [comment]
@rbran100 

namespaced several functions

**Comment by priya-sudarshanam** [comment]
@rbran100 
Edited function names to follow the ossap_stats naming structure

**Comment by priya-sudarshanam** [comment]
@rbran100 
added the necessary google-client files

**Comment by rbran100** [comment]
Closing Old PR

**Pull request closed without merging**